### DISCUSSION TITLE
Feedback on github homepage

### DISCUSSION THREAD

**Discussion by Jonhels** [community/community#124655] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Would it not be better to have a feed of recents commits and updates to repositories i have starred, created or is a member of?
I feel like the homepage right now is only for news and updates of what others are following. Would be really nice to see my issues as well as updates to different repositories.

**Comment by shabir-mp** [comment]
Agree !! Sometimes I need a feature like this. 👍
